collector/library/pod.js

```diff
@@ -46,7 +49,7 @@ class Pod {
 
         this.serviceName = serviceName;
         this.targetUrl = targetUrl;
-        this.checkInterval = checkInterval;
+        this.checkIntervalInMilli = checkInterval;
```

Comment:
This is the time in ms which we will sleep between attempts to handle data from our designated source.
It can be used however in multiple failing scenarios (either logstash not ready, we failed downloading the batch, the endpoint hasn't updated to a boyar with the logs-service)

I want to call it just generally `timeoutInMs` but not sure it's the best  name. I would love your help to try and nail the correct name for this specific number.